tests/test_uv_plots.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import os
+
+
+def test_plot_uv_coverage():
+    from simulations.uv_plots import plot_uv_coverage
+
+    u = np.array(range(5))
+    v = np.array(range(5))
+
+    assert plot_uv_coverage(u, v) is None
```

Comment:
Was genau fragt man mit diesem `is None` ab?

Reply:
The plot function is called. As it does not return anything, we expect a none as result. If there is a problem during plotting an error will be raised and the test fails.